crates/typst-library/src/text/raw.rs

```diff
@@ -218,6 +218,21 @@ pub struct RawElem {
     #[internal]
     #[parse(theme_data.map(Some))]
     pub theme_data: Option<Bytes>,
+
+    /// The size for a tab stop in spaces. A tab is replaced with enough spaces to
+    /// align with the next multiple of the size.
+    ///
+    /// ````example
+    /// #set raw(tab-size: 8)
+    /// ```tsv
+    /// Year	Month	Day
+    /// 2000	2	3
+    /// 2001	2	1
+    /// 2002	3	10
+    /// ```
```

Comment:
in my browser the 3 and the Day doesn't align because the tab size is 4. maybe that's ok and expected, but maybe we can also shorten to just Y, M, D to make this less confusing and show alignment in both the code and the preview?

Reply:
hmm, okay.